toolkit/scss/shared_placeholders/_visuallyhidden.scss

```diff
@@ -0,0 +1,4 @@
+%visuallyhidden {
+  position: absolute;
+  left: -9999em;
```

Comment:
It's a screen reader thing.  Screen readers ignore things that have `display: none;` styling.
https://www.gov.uk/service-manual/user-centred-design/resources/patterns/question-pages.html#accessibility

As far as putting it on the right or left, that seems like a reasonable point.  I was following [the existing style that was in the supplier app](https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/0578507db1d2cfb23b55f177f4d74f1de69bd3ce/app/assets/scss/_single-question-page.scss), but I would by curious to hear @tombye's opinion.